README.md

```diff
@@ -25,8 +25,11 @@ This a mobile web application where you can manage your budget and have a list o
 ![](./app/assets/images/ERD.png)
 
 
+# Loom Video
+https://www.loom.com/share/986104a3758741f89d0808f573bb0c4c
+
```

Comment:
- [ ] kindly add some pieces of code in your video :+1: 
**Note:** The video should contain a highlight about a piece of the code the that you find interesting or is proud of making.